iron-list.html

```diff
@@ -500,9 +503,16 @@
      * @method updateViewportBoundaries
      */
     updateViewportBoundaries: function() {
-      var scrollerStyle = window.getComputedStyle(this._scroller);
-      this._scrollerPaddingTop = parseInt(scrollerStyle['padding-top']);
-      this._viewportSize = this._scroller.offsetHeight;
+      if (this.scroller === window) {
+        this._scrollerPaddingTop = 0;

```

Comment:
Seems like taking `padding-top` from `getComputedStyle('document.body')` would be the equivalent to the else case...?

overflow div with padding: http://jsbin.com/zozafo/edit?html,output

vs.

document scrolling with padding on body: http://jsbin.com/dinomi/edit?html,output